### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by amashadihossein**
- Adds faq to vignette
- Adds local daap vignette
- Adds cmd check badge
- Updates authors
- Adds back cmd check on test
- Adds check to main and dev

### CODE REVIEW

File: vignettes/dp_merge.Rmd [hunk 1]
```diff
@@ -102,23 +99,32 @@ prior to build the `dp_xcars` logic.
 ```{r dpbuild, eval = F}
 dp_connect(board_params = config$board_params, creds = config$creds)
 
+# specify which versions of data products to use
+version1 = "<version1>"
+version2 = "<version2>"
 dp_cars <- dp_get(board_params = config$board_params,
-                  data_name = "dp-cars-us001")
+                  data_name = "dp-cars-us001", version = version1)
```
Comment (amashadihossein) [inline vignettes/dp_merge.Rmd:106]: Note: caching the data before converting it to link is not necessary.

Comment (camorosi) [reply vignettes/dp_merge.Rmd:106]: Would you prefer we take out lines 105-109 and just reference the dp_link instead?

Comment (amashadihossein) [reply vignettes/dp_merge.Rmd:106]: We can take it out now or later as it is benign just unnecessary

Comment (camorosi) [reply vignettes/dp_merge.Rmd:106]: ok, I'll leave in for now

**Review by camorosi — APPROVED** [review]
All looks good to me

**Pull request merged**